Assets/HoloToolkit/Utilities/Editor/ProjectSettingsWindow.cs

```diff
@@ -0,0 +1,274 @@
+								foundBuildTargetVRSettings = true;
+								foundBuildTargetMetro = true;
+								foundBuildTargetEnabled = true;
+								foundDevices = true;
+								foundHoloLens = true;
+							}
+							else
+							{
+								// The target VR settngs were found but the others
+								// still need to be searched for.
+								foundBuildTargetVRSettings = true;
+							}
+						}
+
+					}
+
+					// Look for the build target for Metro
+					else if (!foundBuildTargetMetro)
+					{
+						if (line.Contains("m_BuildTarget: Metro"))
+						{
+							foundBuildTargetMetro = true;
+						}
+
+					}
+
+					else if (!foundBuildTargetEnabled)
+					{
+						if (line.Contains("m_Enabled"))
+						{
+							line = "    m_Enabled: 1";
+							foundBuildTargetEnabled = true;
+						}
+					}
+
+					// Look for the enabled Devices list
+					else if (!foundDevices)
+					{
+						if (line.Contains("m_Devices:"))
+						{
+							// Clear the empty array symbols if any
+							line = line.Replace(" []", "");
+							foundDevices = true;
+						}
+
+					}
+
+					// Once we've found the list look for HoloLens or the next non element
+					else if (!foundHoloLens)
+					{
+						// If this isn't an element in the device list
+						if (!line.Contains("-"))
+						{
+							// add the hololens element, and mark it found
+							builder.Append("    - HoloLens\n");
+							foundHoloLens = true;
+						}
+
+						// Otherwise test if this is the hololens device
+						else if (line.Contains("HoloLens"))
+						{
+							foundHoloLens = true;
+						}
+					}
+
+					builder.Append(line);
+
+					// Write out a \n for all but the last line
+					// NOTE: Specifically preserving unix line endings by avoiding StringBuilder.AppendLine
+					if (i != lines.Length - 1)
+						builder.Append('\n');
+				}
+
+				// Capture the final string
+				settings = builder.ToString();
+
+				File.WriteAllText(settingsPath, settings);
+			}
+			catch (Exception e)
+			{
+				Debug.LogException(e);
+			}
+		}
+
+		/// <summary>
+		/// Modifies the WSA default quality setting to the fastest
+		/// </summary>
+		private void SetFastestDefaultQuality()
+		{
+			try
+			{
+				// Find the WSA element under the platform quality list and replace it's value with 0
+				string settingsPath = "ProjectSettings/QualitySettings.asset";
+				string matchPattern = @"(m_PerPlatformDefaultQuality.*Windows Store Apps:) (\d+)";
+				string replacePattern = @"$1 0";
+
+				string settings = File.ReadAllText(settingsPath);
+				settings = Regex.Replace(settings, matchPattern, replacePattern, RegexOptions.Singleline);
+
+				File.WriteAllText(settingsPath, settings);
+			}
+			catch (Exception e)
+			{
+				Debug.LogException(e);
+			}
+		}
+
+		// Overrides
+		protected override void ApplySettings()
+		{
+			// See the blow notes for why text asset serialization is required
+			if (EditorSettings.serializationMode != SerializationMode.ForceText)
```

Comment:
When in a development environment, where you're regularly committing to shared projects it's best if you have both Serialization and Visible Meta files enabled.

I can only see the usefulness of having binary when building final applications, but generally this is done by either one person, or a automated build agent.